TOC.md

```diff
@@ -109,7 +109,7 @@
     - [Upgrade the Component Version](op-guide/ansible-deployment-rolling-update.md#upgrade-the-component-version)
     - [TiDB 2.0 Upgrade Guide](op-guide/tidb-v2.0-upgrade-guide.md)
     - [TiDB 2.1 Upgrade Guide](op-guide/tidb-v2.1-upgrade-guide.md)
-    - [FAQs After Upgrading](op-guide/upgrade-FAQ.md)
+    - [FAQs After Upgrading](op-guide/upgrade-faq.md)
```

Comment:
Is `FAQs After Upgrade` more appropriate? Generally, we expect "after upgrading to ..." or "after upgrading sth.".
```suggestion
    - [FAQs After Upgrade](op-guide/upgrade-faq.md)
```